File: ios/NeoAirplayViewManager.m
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(NeoAirplayViewManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(tintColor, UIColor *)
RCT_EXPORT_VIEW_PROPERTY(activeTintColor, UIColor *)
RCT_EXPORT_VIEW_PROPERTY(prioritizesVideoDevices, BOOL)


@end

@interface RCT_EXTERN_MODULE(NeoAirplayModule, NSObject)

RCT_EXTERN_METHOD(fetchAVAudioSessionRoutes:
                  (RCTPromiseResolveBlock)resolve
                  reject:(RCTPromiseRejectBlock)reject)

@end

File: ios/NeoAirplayViewManager.swift
@objc(NeoAirplayViewManager)
class NeoAirplayViewManager: RCTViewManager {

  override func view() -> (NeoAirplayView) {
    return NeoAirplayView()
  }

  @objc override static func requiresMainQueueSetup() -> Bool {
    return false
  }
}


